rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/execution/RdReferencedExecution.java

```diff
@@ -0,0 +1,13 @@
+package org.rundeck.app.data.model.v1.execution;
+
+import org.rundeck.app.data.model.v1.job.JobData;
+
+import java.util.List;
+
+public interface RdReferencedExecution {
+
+    JobData getScheduledExecution();
```

Comment:
This should be a reference to the id of the associated job. Using JobData here forces the provider implementation to get the job data along with this record which might be unnecessary, and might be transferring much more data than is actually needed.

Reply:
I changed it